selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/dashboard/organization/MemberOrganizationViewsTest.java

```diff
@@ -10,110 +10,113 @@
 import org.eclipse.che.selenium.pageobject.dashboard.NavigationBar;
 import org.eclipse.che.selenium.pageobject.dashboard.organization.OrganizationListPage;
 import org.eclipse.che.selenium.pageobject.dashboard.organization.OrganizationPage;
-import org.slf4j.Logger;
-import org.slf4j.LoggerFactory;
 import org.testng.annotations.AfterClass;
 import org.testng.annotations.BeforeClass;
 import org.testng.annotations.Test;
 
 /**
- * Test validates organization views for simple user.
+ * Test validates organization views for simple user being a member of any organization.
  *
  * @author Ann Shumilova
  */
-public class UserOrganizationTest {
-  private static final Logger LOG = LoggerFactory.getLogger(UserOrganizationTest.class);
+public class MemberOrganizationViewsTest {
+  private static final String PARENT_ORG_NAME = generate("organization", 5);
+  private static final String CHILD_ORG_NAME = generate("organization", 5);
 
+  private List<OrganizationDto> organizations;
   private OrganizationDto parentOrganization;
-  private OrganizationDto childOrganization;
-
-  @Inject private OrganizationListPage organizationListPage;
-  @Inject private OrganizationPage organizationPage;
-  @Inject private NavigationBar navigationBar;
-  @Inject private Dashboard dashboard;
+  private OrganizationDto subOrganization;
 
   @Inject
   @Named("admin")
   private TestOrganizationServiceClient testOrganizationServiceClient;
 
+  @Inject private OrganizationListPage organizationListPage;
+  @Inject private OrganizationPage organizationPage;
+  @Inject private NavigationBar navigationBar;
+  @Inject private Dashboard dashboard;
   @Inject private TestUser testUser;
 
   @BeforeClass
   public void setUp() throws Exception {
-    parentOrganization =
-        testOrganizationServiceClient.create(NameGenerator.generate("organization", 5));
-    childOrganization =
-        testOrganizationServiceClient.create(
-            NameGenerator.generate("organization", 5), parentOrganization.getId());
+    parentOrganization = testOrganizationServiceClient.create(PARENT_ORG_NAME);
+    subOrganization =
+        testOrganizationServiceClient.create(CHILD_ORG_NAME, parentOrganization.getId());
 
     testOrganizationServiceClient.addMember(parentOrganization.getId(), testUser.getId());
-    testOrganizationServiceClient.addMember(childOrganization.getId(), testUser.getId());
-
+    testOrganizationServiceClient.addMember(subOrganization.getId(), testUser.getId());
+    organizations = testOrganizationServiceClient.getAll();
     dashboard.open(testUser.getName(), testUser.getPassword());
   }
 
   @AfterClass
   public void tearDown() throws Exception {
-    testOrganizationServiceClient.deleteById(childOrganization.getId());
-    testOrganizationServiceClient.deleteById(parentOrganization.getId());
+    for (OrganizationDto organization : organizations)
+      testOrganizationServiceClient.deleteById(organization.getId());
```

Comment:
Such code can affect organizations which are created in parallel threads of test execution.